Formula/influxdb-cli.rb

```diff
@@ -33,6 +34,14 @@ def install
 
     system "go", "build", *std_go_args(ldflags: ldflags),
            "-o", bin/"influx", "./cmd/influx"
+
+    bash_complete = buildpath/"bash-completion"
+    bash_complete.write Utils.safe_popen_read(bin/"influx", "completion", "bash")
+    bash_completion.install bash_complete => "influx"
```

Comment:
Wasn't sure about this: will `install` overwrite any existing completions file on-disk? Thinking ahead to when new versions add new commands, we'd want to make sure they're covered by completions.

Reply:
Anything installed to a filesystem location in an install block will be installed (and overwritten) on updates.

Reply:
I think this is correct as-is, then